src/controller/CHIPDeviceController.cpp

```diff
@@ -95,7 +95,9 @@ ChipDeviceController::ChipDeviceController()
     mDevicePort        = CHIP_PORT;
     mInterface         = INET_NULL_INTERFACEID;
     mLocalDeviceId     = 0;
+    mNumCachedPackets  = 0;
     memset(&mOnComplete, 0, sizeof(mOnComplete));
+    memset(&mCachedPackets, 0, sizeof(mCachedPackets));
```

Comment:
this makes me want a ZEROAT() macro...

Reply:
@rwalker-apple which header will be ideal for adding the macro?

Reply:
good question.  perhaps in a CHIPMem header or a lib support header